packages/foundation/src/slots.tsx

```diff
@@ -191,7 +191,16 @@ function _constructFinalProps<TProps extends IProcessedSlotProps>(defaultStyles:
     assign(finalProps, ...(props as any));
   }
 
-  finalProps.className = mergeStyles(defaultStyles, classNames);
+  if (typeof defaultStyles === 'string') {
+    finalProps.className = defaultStyles;
```

Comment:
Would this work?

```tsx
finalProps.className = css(defaultStyles, ...classNames);
```

Reply:
It might be good to walk through this code in person just to make sure i understand what's going on.

Reply:
For sure, just grab me whenever you have the chance and we can walk through the code together.

Reply:
Does `css` merge in order of precedence? I think that's a concern here.

Reply:
I think it does merge in order of precedence.